SeaSharpe_CVGS/SeaSharpe_CVGS/Migrations/Configuration.cs

```diff
@@ -258,8 +258,8 @@ private Order MakeOrder(string member, string aprover, string orderPlacementDate
             {
                 Member = db.Members.First(m => m.User.UserName == member),
                 Aprover = db.Employees.FirstOrDefault(e => e.User.UserName == aprover),
-                BillingAddress = db.Addresses.OrderBy(c => c.City).Skip(billingAddressIndex).FirstOrDefault(),
-                ShippingAddress = db.Addresses.OrderBy(c => c.City).Skip(shippingAddressIndex).FirstOrDefault(),
+                BillingAddress = db.Addresses.Find(billingAddressIndex),

```

Comment:
Billing address index is not an address id, and the find method is used to find an ID. You should be using skip and firstordefault on a query ordered by the ID. That way if the index is 0 you get the first address and if the index is 1 you get the second.